.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -7,8 +6,9 @@
 
 <!-- Delete this section if this change involves no API changes. -->
 
-Copy [this template](https://github.com/stripe/react-stripe-elements/tree/master/.github/API_REVIEW.md) **or** link to an API review issue.
-
+Copy
+[this template](https://github.com/stripe/react-stripe-elements/tree/master/.github/API_REVIEW.md)
+**or** link to an API review issue.
```

Comment:
This line gets broken up weirdly, so that only `Copy` is on its own line.

Can we use a reference-style link to make it more readable?

```md
Copy [this template] **or** link to an API review issue.

[this template]: https://github.com/stripe/react-stripe-elements/tree/master/.github/API_REVIEW.md
```